VisualPinball.Unity/VisualPinball.Unity/VPT/Trough/TroughComponent.cs

```diff
@@ -219,7 +219,7 @@ public override TroughData CopyDataTo(TroughData data, string[] materialNames, s
 						return new[] {
 							new GamelogicEngineSwitch(EntrySwitchId) { Description = "Entry Switch" }
 						}.Concat(Enumerable.Repeat(0, SwitchCount)
-							.Select((_, i) => new GamelogicEngineSwitch($"ball_switch_{i + 1}", i + 1)
+							.Select((_, i) => new GamelogicEngineSwitch($"ball_switch_{i + 1}")

```

Comment:
Also add a comment so it's clear this is used in the regex.

Reply:
Done!